content/zh/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -0,0 +1,317 @@
+<!--
+---
+reviewers:
+- verb
+- yujuhong
+title: Ephemeral Containers
+content_template: templates/concept
+weight: 80
+---
+-->
+
+---
+reviewers:
+- verb
+- yujuhong
+title: 临时容器
+content_template: templates/concept
+weight: 80
+---
```

Comment:
move line 12-19 to the head of the file;
remove 'reviewers' because the localized content won't be reviewed by those people.